### ISSUE TITLE
Need a Boilerplates starter project to simplify the configuration process

### ISSUE THREAD

**Issue by mathwo** [nvlang/sveltex#1]
This project has great target but the doc is hard to follow without a boilerplate start project for beginners. Thanks!

**Comment by nvlang** [comment]
Thank you for your feedback!

I just published a Node.js package, [`create-sveltex`](https://www.npmjs.com/package/create-sveltex), to try to help with this. You should be able to use it by running

```sh
pnpm dlx create-sveltex # If using PNPM
bunx create-sveltex     # If using Bun
npx create-sveltex      # If using NPM
yarn dlx create-sveltex # If using Yarn
```

Please let me know if this fixes your issue. Alternatively, please feel free to let me know if you have any issues using SvelTeX and I'll do my best to help.
